libs/designsystem/src/lib/components/page/page.component.scss

```
@@ -66,11 +77,42 @@ ion-title {
 }
 
 ion-back-button {
-  --color: #{utils.get-color('black')};
+  --color: #{utils.get-color('white-contrast')};
+  --background: #{utils.get-color('white')};
+  --background-focused: none;
   --icon-font-size: 24px;
 
-  height: utils.$fat-finger-size;
-  width: utils.$fat-finger-size;
+  height: utils.size('xl');
+  width: utils.size('xl');
+
+  // Properly center back-icon
+  &::part(icon) {
+    margin-inline-start: 0;
+    margin-inline-end: 0;
+  }
+
+  &::part(native) {
+    margin: 0 utils.size('xxxs') 0 0;
+    border-radius: utils.$border-radius-round;
+    border: 1px solid transparent;
+    @include utils.focus {
+      border: 1px solid #{utils.$focus-ring-color};
+    }
```

A good rule of thumb is to apply Sass mixins before regular declarations:

```suggestion
  &::part(native) {
    @include utils.focus {
      border: 1px solid #{utils.$focus-ring-color};
    }

    margin: 0 utils.size('xxxs') 0 0;
    border-radius: utils.$border-radius-round;
    border: 1px solid transparent;
```